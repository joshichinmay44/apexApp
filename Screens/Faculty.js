import React, { Component } from 'react'
import { Text, View , ScrollView} from 'react-native'
import styles from '../style/Style'
import {Appbar, Card}  from 'react-native-paper'
export default class Faculty extends Component {
    render() {
        return (
            <View >
              <Appbar.Header>
            <Appbar.Action
              icon="menu"
              onPress={() => this.props.navigation.openDrawer()}
            />
            <Appbar.Content title="Permanent Faculty" />
          </Appbar.Header>
<ScrollView style={styles.Scroll}>
<Card style={styles.cardContainer}>
    <View>
    <Text style={styles.detail}>
        Air Marshal (retd.) Bhushan Gokhale
          </Text>
    </View>
<Card.Cover source={require('../images/Faculty1.jpg')}/>
<Card.Content>
    <Text style={styles.Info}>
    Air Marshal Bhushan Nilkanth Gokhale, PVSM AVSM VM was commissioned into the Indian Air Force in June 1968 in fighter stream.  He has flown over 3500 hours on a variety of combat and trainer aircraft. The Air Marshal has seen active operations during the 1971 Indo-Pak war, on both Eastern and Western fronts.  He is a Qualified Flying Instructor and Fighter Combat Leader from the prestigious Tactics and Combat Development Establishment.
    </Text>
</Card.Content>
</Card>

<Card style={styles.cardContainer}>
    <View>
    <Text style={styles.detail}>
            Lt. Col. (retd) Pradeep Brahmankar
          </Text>
    </View>
<Card.Cover source={require('../images/apex_about.jpg')}/>
<Card.Content>
    <Text style={styles.Info}>
    A Metallurgical Engineering Graduate from the Govt. College of Engineering, Pune (COEP) of the 1976-80 batch. With a three and half year stint in private sector, Lt Col (Retd) Pradeep Brahmankar joined the Indian Military Academy - Dehradun in Feb 1984 and got commissioned on 15 Dec 1984 as ‘Lieutenant’ in the Corps of EME. The officer is from the 58th Technical Graduate Course (TGC) which passed out with 65 NDA / 75 REGULAR Course at Indian Military Academy i.e. IMA - Dehradun. Being a technical graduate, the officer got two years ante-date seniority.
    </Text>
</Card.Content>
</Card>
<Card>

</Card>
</ScrollView>

            </View>
        )
    }
}
