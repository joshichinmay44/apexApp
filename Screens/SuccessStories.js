import React, {Component} from 'react';
import {
  StyleSheet,
  Text,
  View,
  ScrollView,
  Image,
  TouchableOpacity,
} from 'react-native';
import {
  Avatar,
  Card,
  Button,
  Title,
  Paragraph,
  Appbar,
} from 'react-native-paper';
import styles from '../style/Style';
export default class SuccessStories extends React.Component {
  back = () => {
    this.props.navigation.navigate('Home');
  };
  render() {
    return (
      <View style={styles.Container}>
        <ScrollView>
          <Appbar.Header>
          <Appbar.Action
              icon="menu"
              onPress={() => this.props.navigation.openDrawer()}
            />
            <Appbar.Content title="Success Stories" />
          </Appbar.Header>
        </ScrollView>
      </View>
    );
  }
}
