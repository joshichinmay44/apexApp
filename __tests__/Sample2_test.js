import 'react-native';
import React from 'react';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';
import SuccessStories from '../Screens/SuccessStories';
import {TestScheduler} from 'jest';
import Faculty from '../Screens/Faculty';

/* global.fetch = jest.fn(() => new Promise(resolve => resolve()));
jest.mock('react-native-gesture-handler', () => {});
jest.mock('react-native-reanimated', () => {});
jest.mock('@react-navigation/drawer', () => {});
jest.mock('@react-navigation/stack', () => {});
 */
test('renders correctly', () => {
  const snap = renderer.create(<Faculty />).toJSON();
  expect(snap).toMatchSnapshot();
});
