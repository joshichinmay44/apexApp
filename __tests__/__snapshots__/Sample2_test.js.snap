// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View>
  <RCTSafeAreaView
    emulateUnlessSupported={true}
    style={
      Array [
        Object {
          "backgroundColor": "#6200ee",
          "elevation": 4,
          "zIndex": 0,
        },
        Object {
          "shadowColor": "#000000",
          "shadowOffset": Object {
            "height": 3,
            "width": 0,
          },
          "shadowOpacity": 0.24,
          "shadowRadius": 4,
        },
      ]
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#6200ee",
          "elevation": 0,
          "flexDirection": "row",
          "height": 56,
          "marginTop": 0,
          "paddingHorizontal": 4,
        }
      }
    >
      <View
        accessibilityRole="button"
        accessibilityStates={Array []}
        accessible={true}
        focusable={true}
        hitSlop={
          Object {
            "bottom": 6,
            "left": 6,
            "right": 6,
            "top": 6,
          }
        }
        isTVSelectable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "overflow": "hidden",
            },
            Array [
              Object {
                "alignItems": "center",
                "justifyContent": "center",
                "margin": 6,
                "overflow": "hidden",
              },
              Object {
                "borderRadius": 18,
                "height": 36,
                "width": 36,
              },
              undefined,
              undefined,
            ],
          ]
        }
      >
        <View
          style={null}
        >
          <View
            style={
              Array [
                Object {
                  "alignItems": "center",
                  "justifyContent": "center",
                },
                Object {
                  "height": 24,
                  "width": 24,
                },
              ]
            }
          >
            <View
              style={
                Object {
                  "bottom": 0,
                  "left": 0,
                  "opacity": 1,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                  "transform": Array [
                    Object {
                      "rotate": "0deg",
                    },
                  ],
                }
              }
            >
              <Text
                accessibilityElementsHidden={true}
                allowFontScaling={false}
                importantForAccessibility="no-hide-descendants"
                pointerEvents="none"
                style={
                  Array [
                    Object {
                      "color": "#ffffff",
                      "fontSize": 24,
                    },
                    Array [
                      Object {
                        "transform": Array [
                          Object {
                            "scaleX": 1,
                          },
                        ],
                      },
                      Object {
                        "backgroundColor": "transparent",
                      },
                    ],
                    Object {
                      "fontFamily": "Material Design Icons",
                      "fontStyle": "normal",
                      "fontWeight": "normal",
                    },
                    Object {},
                  ]
                }
              >
                Ôçú
              </Text>
            </View>
          </View>
        </View>
      </View>
      <View
        accessible={true}
        focusable={false}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "flex": 1,
              "paddingHorizontal": 12,
            },
            Array [
              Object {
                "marginLeft": 8,
              },
              Object {
                "alignItems": "center",
              },
              undefined,
            ],
          ]
        }
      >
        <Text
          accessibilityRole="header"
          accessibilityTraits="header"
          accessible={true}
          numberOfLines={1}
          style={
            Array [
              Object {
                "color": "#000000",
                "fontFamily": "System",
                "fontWeight": "400",
                "textAlign": "left",
              },
              Array [
                Object {
                  "color": "#ffffff",
                  "fontFamily": "System",
                  "fontWeight": "400",
                },
                Object {
                  "fontSize": 17,
                },
                undefined,
              ],
            ]
          }
        >
          Permanent Faculty
        </Text>
      </View>
      <View
        style={
          Object {
            "width": 48,
          }
        }
      />
    </View>
  </RCTSafeAreaView>
  <RCTScrollView
    style={
      Object {
        "backgroundColor": "ghostwhite",
        "height": "100%",
        "marginBottom": 10,
      }
    }
  >
    <View>
      <View
        style={
          Object {
            "backgroundColor": "#ffffff",
            "borderRadius": 4,
            "borderWidth": 1,
            "elevation": 1,
            "marginBottom": "5%",
            "marginTop": 3,
            "shadowColor": "#000000",
            "shadowOffset": Object {
              "height": 0.5,
              "width": 0,
            },
            "shadowOpacity": 0.24,
            "shadowRadius": 0.75,
            "width": "100%",
          }
        }
      >
        <View
          accessible={true}
          focusable={false}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "flexGrow": 1,
            }
          }
        >
          <View
            index={0}
            siblings={
              Array [
                "View",
                "withTheme(Card.Cover)",
                "Card.Content",
              ]
            }
            total={3}
          >
            <Text
              style={
                Object {
                  "fontFamily": "georgia",
                  "fontSize": 17,
                  "fontWeight": "bold",
                  "marginBottom": "2%",
                  "marginLeft": "2%",
                  "marginTop": "4%",
                }
              }
            >
              Air Marshal (retd.) Bhushan Gokhale
            </Text>
          </View>
          <View
            style={
              Array [
                Object {
                  "backgroundColor": "#eeeeee",
                  "height": 195,
                  "overflow": "hidden",
                },
                undefined,
                undefined,
              ]
            }
          >
            <Image
              siblings={
                Array [
                  "View",
                  "withTheme(Card.Cover)",
                  "Card.Content",
                ]
              }
              source={
                Object {
                  "testUri": "../../../images/Faculty1.jpg",
                }
              }
              style={
                Array [
                  Object {
                    "flex": 1,
                    "height": undefined,
                    "justifyContent": "flex-end",
                    "padding": 16,
                    "resizeMode": "cover",
                    "width": undefined,
                  },
                  undefined,
                ]
              }
            />
          </View>
          <View
            style={
              Array [
                Object {
                  "paddingHorizontal": 16,
                },
                Object {
                  "paddingBottom": 16,
                },
                undefined,
              ]
            }
          >
            <Text
              style={
                Object {
                  "fontFamily": "georgia",
                  "fontSize": 15,
                  "marginTop": "2%",
                }
              }
            >
              Air Marshal Bhushan Nilkanth Gokhale, PVSM AVSM VM was commissioned into the Indian Air Force in June 1968 in fighter stream.  He has flown over 3500 hours on a variety of combat and trainer aircraft. The Air Marshal has seen active operations during the 1971 Indo-Pak war, on both Eastern and Western fronts.  He is a Qualified Flying Instructor and Fighter Combat Leader from the prestigious Tactics and Combat Development Establishment.
            </Text>
          </View>
        </View>
      </View>
      <View
        style={
          Object {
            "backgroundColor": "#ffffff",
            "borderRadius": 4,
            "borderWidth": 1,
            "elevation": 1,
            "marginBottom": "5%",
            "marginTop": 3,
            "shadowColor": "#000000",
            "shadowOffset": Object {
              "height": 0.5,
              "width": 0,
            },
            "shadowOpacity": 0.24,
            "shadowRadius": 0.75,
            "width": "100%",
          }
        }
      >
        <View
          accessible={true}
          focusable={false}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "flexGrow": 1,
            }
          }
        >
          <View
            index={0}
            siblings={
              Array [
                "View",
                "withTheme(Card.Cover)",
                "Card.Content",
              ]
            }
            total={3}
          >
            <Text
              style={
                Object {
                  "fontFamily": "georgia",
                  "fontSize": 17,
                  "fontWeight": "bold",
                  "marginBottom": "2%",
                  "marginLeft": "2%",
                  "marginTop": "4%",
                }
              }
            >
              Lt. Col. (retd) Pradeep Brahmankar
            </Text>
          </View>
          <View
            style={
              Array [
                Object {
                  "backgroundColor": "#eeeeee",
                  "height": 195,
                  "overflow": "hidden",
                },
                undefined,
                undefined,
              ]
            }
          >
            <Image
              siblings={
                Array [
                  "View",
                  "withTheme(Card.Cover)",
                  "Card.Content",
                ]
              }
              source={
                Object {
                  "testUri": "../../../images/apex_about.jpg",
                }
              }
              style={
                Array [
                  Object {
                    "flex": 1,
                    "height": undefined,
                    "justifyContent": "flex-end",
                    "padding": 16,
                    "resizeMode": "cover",
                    "width": undefined,
                  },
                  undefined,
                ]
              }
            />
          </View>
          <View
            style={
              Array [
                Object {
                  "paddingHorizontal": 16,
                },
                Object {
                  "paddingBottom": 16,
                },
                undefined,
              ]
            }
          >
            <Text
              style={
                Object {
                  "fontFamily": "georgia",
                  "fontSize": 15,
                  "marginTop": "2%",
                }
              }
            >
              A Metallurgical Engineering Graduate from the Govt. College of Engineering, Pune (COEP) of the 1976-80 batch. With a three and half year stint in private sector, Lt Col (Retd) Pradeep Brahmankar joined the Indian Military Academy - Dehradun in Feb 1984 and got commissioned on 15 Dec 1984 as ‚ÄòLieutenant‚Äô in the Corps of EME. The officer is from the 58th Technical Graduate Course (TGC) which passed out with 65 NDA / 75 REGULAR Course at Indian Military Academy i.e. IMA - Dehradun. Being a technical graduate, the officer got two years ante-date seniority.
            </Text>
          </View>
        </View>
      </View>
      <View
        style={
          Object {
            "backgroundColor": "#ffffff",
            "borderRadius": 4,
            "elevation": 1,
            "shadowColor": "#000000",
            "shadowOffset": Object {
              "height": 0.5,
              "width": 0,
            },
            "shadowOpacity": 0.24,
            "shadowRadius": 0.75,
          }
        }
      >
        <View
          accessible={true}
          focusable={false}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "flexGrow": 1,
            }
          }
        />
      </View>
    </View>
  </RCTScrollView>
</View>
`;
